pkg/fanal/analyzer/sbom/sbom.go

```diff
@@ -83,3 +71,21 @@ func (a sbomAnalyzer) Type() analyzer.Type {
 func (a sbomAnalyzer) Version() int {
 	return version
 }
+
+func handleBitnamiImages(componentPath string, bom types.SBOM) {
+	for i, app := range bom.Applications {
+		if app.Type == ftypes.Bitnami {
+			// Set the component dir path to the application
+			bom.Applications[i].FilePath = componentPath
+			continue
```

Comment:
Why this continue? I mean, why not iterating through `app.Libraries` when type is `bitnami`?

Reply:
Added a comment.
https://github.com/aquasecurity/trivy/pull/5062/commits/1f68bd51cd6472d7b0d0fddd8d2651c5869091b7

`Application` in Trivy represents a group of packages, such as package-lock.json, like a project. It doesn't make sense to set file paths to all the packages in package-lock.json. Bitnami SPDX aggregates packages and doesn't have the respective file paths now. Then, we just set file paths to the application level.

However, artifacts like JAR files are located in the image separately. They all have different file paths, and those file paths are set to `Application.Libraries[].FilePath` in Trivy.

In short,

- Application.FilePath: a file path for aggregated packages
- Application.Libraries[].FilePath: a file path for artifacts

Ideally, we should determine that from `Application.Type`, but I put that in our backlog. I know the design is not straightforward now, so I'm planning to refactor it.

It is in its early stages. We can improve this feature step by step.